batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/dao/EmployeeDao.java

```diff
@@ -0,0 +1,137 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Employee;
+import com.lftechnology.batch7crud.util.DBConnection;
+
+import javax.xml.crypto.Data;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.xml.crypto.Data'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)